vendor/modules.txt

```diff
@@ -1,74 +1,55 @@
 # github.com/bmatsuo/lmdb-go v1.6.1-0.20160816100615-69ad631904c9
-## explicit
```

Comment:
@oleorhagen Why did my `go mod vendor` call removed all these? Should I manually revert it?

Reply:
No idea.
My guess, a difference between `1.14` and `1.15`?